docs-source/content/userguide/managing-domains/domain-lifecycle/restarting.md

```diff
@@ -3,116 +3,123 @@ title: "Restarting"
+For detailed information on how to restart servers using the operator, see [Starting, stopping, and restarting servers]({{< relref "/userguide/managing-domains/domain-lifecycle/startup/_index.md" >}}).
 
 #### Common restart scenarios
 
 This document describes what actions you need to take to properly restart your servers for a number of common scenarios:
 
-* Modifying the WebLogic configuration
-* Changing the custom domain configuration overrides (also called situational configuration) for Domain in PV and Domain in Image domains
+* Modifying the WebLogic domain configuration
+* Changing the domain configuration overrides (also called situational configuration) for Domain in PV and Domain in Image domains
 * Changing the model files for Model in Image domains
 * Changing the WebLogic Server credentials (the user name and password)
-* Changing properties on the domain resource that affect server pods (such as `image`, `volumes`, and `env`)
+* Changing fields on the Domain that [affect WebLogic Server instance Pod generation]({{< relref "/userguide/managing-domains/domain-lifecycle/startup/_index.md#fields-that-cause-servers-to-be-restarted" >}}) (such as `image`, `volumes`, and `env`)
 * Applying WebLogic Server patches
-* Updating deployed applications for domain home in image
+* Updating deployed applications for Domain in Image
 
 ### Use cases
 
-#### Modifying the WebLogic Server configuration
+#### Modifying the WebLogic domain configuration
 
-Changes to the Oracle WebLogic Server configuration may require either a rolling or full domain restart depending on the domain home location and the type of configuration change.
+Changes to the WebLogic domain configuration may require either a rolling or full domain restart depending on the domain home location and the type of configuration change.
 
 * **Domain in Image:**
-For a domain home in image, any changes (dynamic or non-dynamic) to the WebLogic configuration requires a full domain restart.  
-    * If you create a new image with a new name, then you must avoid a rolling restart, which can cause unexpected behavior for the running domain due to configuration inconsistencies as seen by the various servers, by following the steps in [Avoiding a rolling restart when changing image property on a domain resource](#avoiding-a-rolling-restart-when-changing-image-property-on-a-domain-resource).
-    * If you create a new image with the same name, then you must manually initiate a full domain restart. See [Full domain restarts]({{< relref "/userguide/managing-domains/domain-lifecycle/startup/_index.md#full-domain-restarts" >}}).
+
+For Domain in Image, you may only perform a rolling restart if both the WebLogic configuration changes between the present image and a new image are dynamic and you have [followed the CI/CD guidelines]({{< relref "/userguide/cicd/mutate-the-domain-layer">}}) to create an image with compatible encryption keys.
```

Comment:
Indentation issue? This paragraph and the following don't appear to be indented as far as their owning bullet.  I could be wrong, as I didn't check in hugo, but the github markup parser is having an issue.